conformance/lang/expressions/relational-expr/greater_than_expression.balt

```diff
@@ -0,0 +1,1253 @@
+   io:println(c); // @output 12
+}
+
+Test-Case: output
+Description: Test greater than expression with list as operands.
+Labels: array-type, Decimal, DecimalFloatingPointNumber, DecimalNumber, int, float, list-constructor-expr, nil-literal, relational-expr, string
+
+function init() {
+   int[] a = [12, 100, 2000000];
+   int[] b = [13, 3];
+   int[] c = [];
+   int[] d = [];
+   string?[] e = ["Hello", "World!", (), "to", "Ballerina"];
+   string?[] f = ["Hello", "World!", (), "from", "Ballerina"];
+   float?[] g = [12.5, 43, ()];
+   float?[] h = [12.5, 43, 4];
+   decimal?[] i = [45, 23.54232, 44];
+   decimal?[] j = [45, 23.54232, 44, ()];
+
+   io:println(a > b); // @output false
+   io:println(c > d); // @output false
+   io:println(c > b); // @output false
+   io:println(h > g); // @output false
+   io:println(i > j); // @output false
+}
+
+Test-Case: output
+Description: Test greater than expression with tuple as operands.
+Labels: Decimal, DecimalFloatingPointNumber, DecimalNumber, int, float, list-constructor-expr, nil-literal, relational-expr, string, tuple-type
+
+function init() {
+   [int, string, float] a = [1, "AB", 33.33];
+   [int, string, float] b = [1, "AB", 34];
+
+   io:println(b > a); // @output true
+
+   [string, string, int?, string] c = ["Hello", "World!", (), "to"];
+   [string, string, int?, string] d = ["Hello", "World!", (), "from"];
+
+   io:println(c > d); // @output true
+
+   [int, float] e = [59215, (0.0 / 0.0)];
+   [int, float] f = [59215, 123.432];
+
+   io:println(e > f); // @output false
+
+   [int, decimal?] g = [59215, 4354.323];
+   [int, decimal?] h = [59215, ()];
+
+   io:println(g > h); // @output false
+}
+
+Test-Case: error
+Description: Test the static type of operands belonging to non ordered type with respect to one operand resulting in an error
+           in greater than expression.
+Labels: array-type, boolean, decimal, float, int, module-type-defn, relational-expr, string
+
+type Strings "A"|"B";
+type Ints int:Unsigned32|int:Signed8;
+type Floats 1.5|2.4;
+type Decimals 3.54|2;
+
+function errorFunction(string a, int b, float c, decimal d, boolean e, int[] f, float[] g, Strings h, Ints i, Floats j, Decimals k) {
+   boolean _ = a > b; // @error > is not allowed with operands of string and int types
+   boolean _ = b > c; // @error > is not allowed with operands of float and int types
+   boolean _ = c > d; // @error > is not allowed with operands of float and decimal types
+   boolean _ = b > e; // @error > is not allowed with operands of int and boolean types
+   boolean _ = f > g; // @error > is not allowed with operands of int[] and int float[]
+   boolean _ = h > i; // @error > is not allowed with operands of user defined string types and int types
+   boolean _ = i > j; // @error > is not allowed with operands of user defined int types and float types
+   boolean _ = j > k; // @error > is not allowed with operands of user defined float types and decimal types
```

Comment:
Shall we also check this with a union like `int|string` as well?.
Let's add for other operations as well.